pkg/virt-handler/virtwrap/schema.go

```diff
@@ -85,6 +108,7 @@ type Domain struct {
 
 type DomainStatus struct {
 	Status LifeCycle
+	Reason LifeCycle
```

Comment:
Mostly duplication of strings ('crashed' is lifecycle and reason, so is 'shutdown').

Splitting them requires another name (like ReasonCrashed, ReasonShutdown) for the duplicated constants. I don't mind either though.